skyportal/plot.py

```diff
@@ -1472,20 +1577,80 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
         margin=(0, 10, 0, 10),
     )
 
+    name_line_list = list(SPEC_LINES.items())
+
     for i, (wavelengths, color) in enumerate(SPEC_LINES.values()):
+
         el_data = pd.DataFrame({'wavelength': wavelengths})
-        obj_redshift = 0 if obj.redshift is None else obj.redshift
-        el_data['x'] = el_data['wavelength'] * (1.0 + obj_redshift)
-        model_dict[f'el{i}'] = plot.segment(
-            x0='x',
-            x1='x',
-            # TODO change limits
-            y0=0,
-            y1=1e4,
-            color=color,
-            source=ColumnDataSource(el_data),
-        )
-        model_dict[f'el{i}'].visible = False
+
+        if name_line_list[i][0] == 'Sky Lines':  # No redshift correction of skylines
+            el_data['x'] = el_data['wavelength']
+            # model_dict is used for collecting all the spec lines which we want to move with redshift and vexp
+            # we defined earlier "sz_dict" for lines (sky, tellurics) which should not move with z and v_exp
+            sz_dict[f'el{i}'] = plot.segment(
+                x0='x',
+                x1='x',
+                y0=0,
+                y1=1e4,
+                color=color,
+                source=ColumnDataSource(el_data),
+            )
+
+            sz_dict[f'el{i}'].visible = False
+
+        elif name_line_list[i][0] == 'Tellurics-1':
+
+            el_data['x'] = el_data['wavelength']
+            # [6867, 6884]
+            midtel1 = (el_data['x'][0] + el_data['x'][1]) / 2
+            widtel1 = el_data['x'][1] - el_data['x'][0]
+
+            sz_dict[f'el{i}'] = plot.vbar(
+                x=midtel1,
+                width=widtel1,
+                # TODO change limits
+                top=1e4,
+                color=color,
+                alpha=0.3,
+                # source=ColumnDataSource(tel1),
+            )
+
+            sz_dict[f'el{i}'].visible = False
+
+        elif name_line_list[i][0] == 'Tellurics-2':
+
+            el_data['x'] = el_data['wavelength']
+            # [7594, 7621]
+
+            midtel2 = (el_data['x'][0] + el_data['x'][1]) / 2
+            widtel2 = el_data['x'][1] - el_data['x'][0]
+
+            sz_dict[f'el{i}'] = plot.vbar(
+                x=midtel2,
+                width=widtel2,
+                # TODO change limits
+                top=1e4,
+                color=color,
+                alpha=0.3,
+                # source=ColumnDataSource(tel2),
+            )
+
+            sz_dict[f'el{i}'].visible = False
```

Comment:
Looks almost identical, how about:

```suggestion
        elif name_line_list[i][0] in ('Tellurics-1', 'Tellurics-2'):

            el_data['x'] = el_data['wavelength']
            midtel1 = (el_data['x'][0] + el_data['x'][1]) / 2
            widtel1 = el_data['x'][1] - el_data['x'][0]

            sz_dict[f'el{i}'] = plot.vbar(
                x=midtel1,
                width=widtel1,
                # TODO change limits
                top=1e4,
                color=color,
                alpha=0.3,
            )

            sz_dict[f'el{i}'].visible = False
```